frontend/viewer/src/lib/entry-editor/field-editors/FieldEditor.svelte

```diff
@@ -35,12 +38,12 @@
 </script>
 
 {#if isMultiString(value)}
-  <MultiFieldEditor on:change id={field.id} wsType={field.ws} bind:value {readonly} />
+  <!-- <MultiWsInput on:change id={field.id} wsType={field.ws} bind:value {readonly} /> -->
 {:else if isSingleString(value)}
-  <SingleFieldEditor on:change id={field.id} wsType={field.ws} bind:value {readonly} />
+  <!-- <WsInput on:change id={field.id} wsType={field.ws} bind:value {readonly} /> -->
 {:else if isSingleOption(state)}
-  <!-- <SingleOptionEditor on:change id={state.field.id} wsType={state.field.ws} bind:value={state.value} {readonly} /> -->
+  <!-- <Select on:change id={state.field.id} wsType={state.field.ws} bind:value={state.value} {readonly} /> -->
 {:else if isMultiOption(state)}
-  <!-- <MultiOptionEditor on:change id={state.field.id} wsType={state.field.ws} bind:value={state.value} {readonly} /> -->
+  <!-- <MultiSelect on:change id={state.field.id} wsType={state.field.ws} bind:value={state.value} {readonly} /> -->
```

Comment:
Are these commented-out lines of code supposed to come back at some point, or should they be removed?

Reply:
It's fairly likely that they'll get removed eventually. This file might add a little bit of value when we get to custom fields 🤷